frontend/logger.lua

```diff
@@ -50,7 +50,15 @@ local function log(log_lvl, dump_lvl, ...)
         end
     end
     if isAndroid then
-        android.LOGI(LOG_PREFIX[log_lvl]..line)
+        if log_lvl == 'dbg' then
```

Comment:
I overlooked this before, but let's stick to double quotes for code consistency?